src/main/java/com/minecolonies/items/ItemBuildTool.java

```diff
@@ -27,13 +27,12 @@ public ItemBuildTool()
     @Override
     public EnumActionResult onItemUse(ItemStack stack, EntityPlayer playerIn, World worldIn, BlockPos pos, EnumHand hand, EnumFacing facing, float hitX, float hitY, float hitZ)
     {
-        if (!worldIn.isRemote)
+        if (worldIn.isRemote)
         {
-            return EnumActionResult.FAIL;
+            playerIn.addStat(ModAchievements.achievementWandOfbuilding);
```

Comment:
If we move the achievement gain inside a `worldIn.isRemote` then on a remote server the achievement won't be awarded.